src/constraint_system/composer.rs

```diff
@@ -994,7 +1001,8 @@ mod tests {
         // So pre-fetch these before calling Prove
         let public_inputs = prover.cs.into_dense_public_inputs();
 
-        (prover.prove(&ck).unwrap(), public_inputs);
+        prover.prove(&ck).unwrap();
+        drop(public_inputs);
```

Comment:
`public_inputs` seems to already drop at the end of this scope. Why drop it explicitly?